pkg/integrations/v4/logs/cfg.go

```diff
@@ -278,7 +299,11 @@ func NewFBConf(loggingCfgs LogsCfg, logFwdCfg *config.LogForward, entityGUID, ho
 	var fbOSConfig FBOSConfig
 	addOSDependantConfig(&fbOSConfig)
 
-	for _, block := range loggingCfgs {
+	totalFiles := 0
+	for i := 0; i < len(loggingCfgs); i++ {
```

Comment:
You don't need to change the "for" shape here, simply include "i" now:
```
for i, block := range loggingCfgs {
// no need to do block := loggingCfgs[i] now
```

Reply:
Thanks for the suggestion! 🙂
I am not aware of this way!